backend/clickhouse/querybuilder.go

```diff
@@ -228,14 +229,19 @@ func parseColumnRule(admin *model.Admin, rule Rule, projectId int, sb *sqlbuilde
 			if !found {
 				return "", fmt.Errorf("separator not found for between query %s", v)
 			}
-			start, err := strconv.ParseInt(before, 10, 64)
+			start, err := strconv.ParseFloat(before, 64)
 			if err != nil {
 				return "", err
 			}
-			end, err := strconv.ParseInt(after, 10, 64)
+			start *= 1000
+			end, err := strconv.ParseFloat(after, 64)
 			if err != nil {
 				return "", err
 			}
+			end *= 1000
+			if after == "60" {
```

Comment:
what's this hard-coded condition for?

Reply:
added comment - this condition exists for opensearch, if the slider is all the way to the right (at 60 mins) we should allow any greater length as well